packages/near-membrane-base/src/blue.ts

```diff
@@ -104,7 +102,7 @@ export function blueProxyFactory(env: MembraneBroker) {
     ) {
         const normalizedRedDescriptors = ObjectGetOwnPropertyDescriptors(originalTarget);
         const targetKeys = ReflectOwnKeys(normalizedRedDescriptors);
-        const blueDescriptors = ObjectCreate(null);
+        const blueDescriptors: PropertyDescriptorMap = { __proto__: null } as any;
```

Comment:
☝️ The type won't allow initializing without `any`?

Reply:
I am guessing it is because of the null value?

Reply:
@jdalton unfortunately no (my understanding matches @garychangsf)

Reply:
@rwaldron Catching up: why was this necessary?

Reply:
> @rwaldron Catching up: why was this necessary?

It allows for declaring a null prototype in the object literal, side stepping the need to create a null object first and then bolting on properties. Dropping `ObjectCreate` and `ObjectAssign` for `__proto__` and object spread.

Reply:
Got it, it's just that every time a see `__foo__` it kinda feels like using low level and potentially unsupported features.